hedera-evm/src/main/java/com/hedera/services/contracts/gascalculator/GasCalculatorHederaV18.java

```diff
@@ -1,126 +0,0 @@
-package com.hedera.services.contracts.gascalculator;
-
-/*
- * -
- * ‌
- * Hedera Services Node
- * ​
- * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
- * ​
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *       http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- * ‍
- *
- */
-
-import com.hedera.services.context.properties.GlobalDynamicProperties;
-import com.hedera.services.fees.HbarCentExchange;
-import com.hedera.services.fees.calculation.UsagePricesProvider;
-import org.apache.tuweni.bytes.Bytes;
-import org.hyperledger.besu.datatypes.Wei;
-import org.hyperledger.besu.evm.account.Account;
-import org.hyperledger.besu.evm.frame.MessageFrame;
-import org.hyperledger.besu.evm.gascalculator.PetersburgGasCalculator;
-
-import javax.inject.Inject;
-
-/**
- * Provides Hedera adapted gas cost lookups and calculations used during transaction processing.
- * Maps to the gas costs of the Smart Contract Service up until 0.18.0 release
- */
-//TODO: replace UsagePricesProvider and HbarCentExchange with an interface
-public class GasCalculatorHederaV18 extends PetersburgGasCalculator {
```

Comment:
If we delete a gas calculator from the evm module, we can no longer execute historical transactions from that gas schedule.  While this may be true of the older transactions eventually we will need to keep all historical gas schedules covering all the transactions we will re-execute.